tasks/bug_report_template.rb

```diff
@@ -12,12 +12,23 @@
   end
 
   # Change Rails version if necessary.
-  gem "rails", "6.1.4"
-
-  gem "sprockets", "3.7.2"
-  gem "sassc-rails", "2.1.2"
-  gem "sqlite3", "1.4.2", platform: :mri
-  gem "activerecord-jdbcsqlite3-adapter", "61.0", platform: :jruby
+  gem "rails", "~> 7.0.0"
+
+  gem "sprockets", "~> 3.7"
```

Comment:
`3.7` is still needed because 4.0 will ask for a manifest file, and I do not know how to avoid that